docs/editor_manual/new_pages/required_fields.md

```diff
@@ -1,10 +1,7 @@
 # Required fields
 
 -   Fields marked with an asterisk are required. You will not be able to save a draft or submit the page for moderation without these fields being completed.
-
-![](../../_static/images/screen22_required_field.png)
```

Comment:
Removed because I couldn’t find a good example of a required field in bakerydemo! The only ones seem to be either within StreamField or the Promote tab, which feel odd to introduce so early on. I thought this would be ok to cover without visuals anyway.